components/docs-chef-io/content/automate/ha_aws_cert_mngr.md

```diff
@@ -59,14 +59,47 @@ Follow these steps if you want to use the AWS certificates in your network infra
 
 1. Log in as a Sudo user by using `su-` command.
 
-2. Navigate to the `hab` workspace directory.
-
-3. Open the `config.toml` file in any editor of your choice.
-
-{{< figure src="/images/automate/ha_load_balancer.png" alt="AWS Certificate ARN Value">}}
-
-4. Copy the existing or generated ARN value from AWS console to the `chef_server_lb_certificate_arn` and `automate_server_lb_certificate_arn` fields in the *config.toml* file.
-
-{{< figure src="/images/automate/ha_a2rb_lb_certificate.png" alt="AWS Certificate ARN Value">}}
-
-5. Save and close the file.
+1. Navigate to the `hab` workspace directory.
+
+1. Open the `config.toml` file in any editor of your choice.
+
+    {{< figure src="/images/automate/ha_load_balancer.png" alt="AWS Certificate ARN Value">}}
+
+1. Copy the existing or generated ARN value from AWS console to the `chef_server_lb_certificate_arn` and `automate_server_lb_certificate_arn` fields in the *config.toml* file.
+
+    ```bash
+        # Provide vpcid and cider block
+        # E.g. aws_vpc_id = “vpc12318h”
+        # E.g. aws_cidr_block_addr = “172.31.64.0”
+        aws_vpc_id = “vac-c2011ba6”
+        aws_cidr_block_addr = “172.31.128.0”
+        # ssh key pair name in AWS to access instances
+        ssh_key_pair_name = “meetg-sydney”
+        ami_filter_name = “ “
+        ami_filter_virt_type = “ “
+        ami_filter_owner = “ “
+        ami_id = “ “
+        automate_server_instance_type = “t3.medium”
+        chef_server_instance_type = “t3.medium”
+        elasticsearch_server_instance_type = “m5.large”
+        postgresql_server_instance_type = “t3.medium”
+        automate_lb_certificate_arn = “areas:am:a-southeast-2:112758395563:certificate/9b9fcc04-6513-4ac5-9332-26a59a6ce4e”
+        chef_server_lb_certificate_arn = “areas:am:a-southeast-2:112758395563:certificate/9b9fcc04-6513-4ac5-9932-262a59a6ce4e”
+        automate_ebs_volume_iops = “100”
+        automate_ebs_volume_size = “50”
+        automate_ebs_volume_type = “gp2”
+        chef_ebs_volume_iops = “100”
+        chef_ebs_volume_size = “50”
+        chef_ebs_volume_type = “gp2”
+        elasticsearch_ebs_volume_iops = “100”
+        elasticsearch_ebs_volume_size = “50”
+        elasticsearch_ebs_volume_type = “gp2”
+        postgresql_ebs_volume_iops = “100”
+        postgresql_ebs_volume_size = “50”
+        postgresql_ebs_volume_type = “gp2”
+        X-Contact = “ “
+        X-Dept = “ “
+        X-Project = “ “
+    ```
```

Comment:
It will be a bit simpler to just include the referenced parameters.

````suggestion
1. Copy and paste the AWS Certificate Manager ARN into the `chef_server_lb_certificate_arn` and `automate_server_lb_certificate_arn` fields in the `config.toml` file. For example:

    ```text
    automate_lb_certificate_arn = “areas:am:a-southeast-2:112758395563:certificate/9b9fcc04-6513-4ac5-9332-26a59a6ce4e”
    chef_server_lb_certificate_arn = “areas:am:a-southeast-2:112758395563:certificate/9b9fcc04-6513-4ac5-9932-262a59a6ce4e”
    ```
````